### PR TITLE
Added demo

### LINKED ISSUE #59: Better readme
- [ ] table of content
- [ ] demo to show how the solution works
- [ ] diagram to show how diff components work together

### PR DESCRIPTION
**Pull request by HoussemDellai**
What this PR does / why we need it: Added a link for a demo video that shows how to install the different component: AAD Pod Identity and Secure Storage CSI driver for Key Vault, then how to use it from a Pod in AKS.

Which issue(s) this PR fixes (optional, using fixes #<issue number>(, fixes #<issue_number>, ...) format, will close the issue(s) when the PR gets merged):
Fixes #59 (demo to show how the solution works)

Special notes for your reviewer: I'm also working on a detailed article :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,9 @@ Azure Key Vault provider for [Secret Store CSI driver](https://github.com/kubern
 
 ## Demo
```
Comment (ritazh) [inline README.md:7]: Thanks for the PR @HoussemDellai 🎉  This is awesome!

Do you mind creating a section `Presentations` at the end of this readme and move this content to that section instead?

Comment (HoussemDellai) [reply README.md]: Hi Rita :)
Done 👍

**Comment by ritazh** [comment]
@HoussemDellai looks like your latest merge may have overwritten your changes.

**Comment by HoussemDellai** [comment]
Now fixed :)

**Comment by HoussemDellai** [comment]
Thank you so much Rita

**Pull request merged**